skops/io/tests/test_audit.py

```diff
@@ -151,9 +151,9 @@ def test_complex_pipeline_untrusted_set():
     clf = Pipeline([
         ("features", FeatureUnion([
             ("scaler", StandardScaler()),
-            ("sqrt", FunctionTransformer(
-                    func=np.sqrt,
-                    inverse_func=np.square,
+            ("np.funcs", FunctionTransformer(
+                    func=np.split,
+                    inverse_func=np.angle,
```

Comment:
This doesn't actually make sense now, does it? We can probably remove this case here, and bellow we have tests to make sure ufuncs are trusted. WDYT @BenjaminBossan

Reply:
Yes, I think this can be removed, as other tests cover what this covers.